### PR TITLE
Merged Vietnamase translations to openhie-hiv-casereport

### LINKED ISSUE #161: Add the openhie hiv ig case report questionnaire to the gallery app
https://openhie.github.io/hiv-ig/Questionnaire-hiv-case-report-questionnaire.json.html

### PR DESCRIPTION
**Pull request by aditya-07**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

Fixes #161

**Description**
Merged Vietnamase translations from questionnaire-case-reporting-questionnaire-vn.json to openhie-hiv-case-report.json as the former was just a subset of the openhie-hiv-case-report.json .

**Alternative(s) considered**
Have you considered any alternatives? And if so, why have you chosen the approach in this PR?

**Type**
Choose one: Other

**Screenshots (if applicable)**

**Checklist**
- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/docs/CODE_OF_CONDUCT.md)
- [x] I have read [How to Contribute](https://github.com/google/android-fhir/blob/master/docs/contributing.md)
- [x] I have read the [Developer's guide](https://github.com/google/android-fhir/wiki/Developer's-Guide)
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate )
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally
- [x] I have built and run the reference app(s) to verify my change fixes the issue and/or does not break the reference app(s)

### CODE REVIEW

File: datacapturegallery/src/main/assets/openhie-hiv-case-report.json [hunk 1]
```diff
@@ -107,56 +107,209 @@
               "linkId": "arv_patient_nr",
               "prefix": "1",
               "text": "ARV Patient Number",
+              "_text": {
```
Comment (jingtang10) [inline datacapturegallery/src/main/assets/openhie-hiv-case-report.json:110]: looking at https://openhie.github.io/hiv-ig/Questionnaire-hiv-case-report-questionnaire.json.html, the prefixes have gone? can you make sure we adhere to the latest version of the questionnaire? thanks

File: datacapturegallery/src/main/java/com/google/android/fhir/datacapture/gallery/QuestionnaireListFragment.kt [hunk 2]
```diff
@@ -97,10 +97,11 @@ class QuestionnaireListFragment : Fragment() {
           ),
           // Example taken from
           // https://github.com/google/android-fhir/issues/161#issuecomment-842540918
+          // and merged to openhie-hiv-case-report.json
           QuestionnaireListItem(
             "HIV Case Report With Vietnamese Translation",
             "HIV Case Reporting and Monitoring IG with Vietnamese Translation",
-            "questionnaire-case-reporting-questionnaire-vn.json"
+            "openhie-hiv-case-report.json"
           )
```
Comment (jingtang10) [inline datacapturegallery/src/main/java/com/google/android/fhir/datacapture/gallery/QuestionnaireListFragment.kt:105]: Just remove this. and add to line 61 that there is vietnamese translation.

**Review by jingtang10 — CHANGES_REQUESTED** [review]

File: datacapturegallery/src/main/assets/openhie-hiv-case-report.json (hunk 1 above)
Comment (aditya-07) [reply datacapturegallery/src/main/assets/openhie-hiv-case-report.json:107]: Done

File: datacapturegallery/src/main/java/com/google/android/fhir/datacapture/gallery/QuestionnaireListFragment.kt (hunk 2 above)
Comment (aditya-07) [reply datacapturegallery/src/main/java/com/google/android/fhir/datacapture/gallery/QuestionnaireListFragment.kt]: Done

File: datacapturegallery/src/main/java/com/google/android/fhir/datacapture/gallery/QuestionnaireListFragment.kt [hunk 3]
```diff
@@ -62,7 +62,7 @@ class QuestionnaireListFragment : Fragment() {
           // https://openhie.github.io/hiv-ig/Questionnaire-hiv-case-report-questionnaire.json.html
           QuestionnaireListItem(
             "HIV Case Report",
-            "HIV Case Reporting and Monitoring IG",
+            "HIV Case Reporting and Monitoring IG along with Vietnamese translations.",
```
Comment (jingtang10) [inline datacapturegallery/src/main/java/com/google/android/fhir/datacapture/gallery/QuestionnaireListFragment.kt:65]:
```suggestion
            "HIV Case Reporting and Monitoring IG with Vietnamese translations",
```

**Review by jingtang10 — APPROVED** [review]
Thanks Aditya!

**Pull request merged**